Frameworks/UIKit/UIWindow.mm

```diff
@@ -372,39 +341,51 @@ - (void)setScreen:(UIScreen*)screen {
 - (void)dealloc {
     [CATransaction _removeLayer:[self layer]];
     [static_cast<NSMutableArray*>([[UIApplication sharedApplication] windows]) removeObject:self];
-
-    [super dealloc];
 }
 
 /**
  @Status Stub
 */
 - (CGPoint)convertPoint:(CGPoint)point toWindow:(UIWindow*)window {
     UNIMPLEMENTED();
-    return StubReturn();
+    return point;
```

Comment:
either's fine - i'll switch it back though

---
In reply to: [111610178](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r111610178) [](ancestors = 111610178)